.github/workflows/compiler-sanitizers.yml

```diff
@@ -23,27 +23,39 @@ jobs:
       with:
         fetch-depth: 0
 
-    - name: Install dependencies
+    - name: Install Dependencies
+      # Set env variable or otherwise tzdata package requires interaction
+      env:
+        DEBIAN_FRONTEND: noninteractive
+      run: |
+        sudo apt update
+        sudo apt install -y \
+            build-essential \
```

Comment:
I wonder if there is a way to externalize this list of packages and share it across all yaml files